manim/utils/bezier.py

```diff
@@ -1123,156 +1118,496 @@ def match_interpolate(
+        system of equations for the 1st handles (example for :math:`N = 5`):
+
+        .. math::
+            \begin{pmatrix}
+                4 & 1 & 0 & 0 & 1 \\
+                1 & 4 & 1 & 0 & 0 \\
+                0 & 1 & 4 & 1 & 0 \\
+                0 & 0 & 1 & 4 & 1 \\
+                1 & 0 & 0 & 1 & 4
+            \end{pmatrix}
+            \begin{pmatrix}
+                H_{1,0} \\
+                H_{1,1} \\
+                H_{1,2} \\
+                H_{1,3} \\
+                H_{1,4}
+            \end{pmatrix}
+            =
+            \begin{pmatrix}
+                4A_0 + 2A_1 \\
+                4A_1 + 2A_2 \\
+                4A_2 + 2A_3 \\
+                4A_3 + 2A_4 \\
+                4A_4 + 2A_5
+            \end{pmatrix}
+
+        which will be expressed as :math:`RH_1 = D`.
+
+        :math:`R` is almost a tridiagonal matrix, so we could use Thomas' algorithm.
+
+        .. seealso::
+            `Tridiagonal matrix algorithm. Wikipedia. <https://en.wikipedia.org/wiki/Tridiagonal_matrix_algorithm>`_
+
+        However, :math:`R` has ones at the opposite corners. A solution to this is
+        the first decomposition proposed in the link below, with :math:`\alpha = 1`:
+
+        .. seealso::
+            `Tridiagonal matrix algorithm # Variants. Wikipedia. <https://en.wikipedia.org/wiki/Tridiagonal_matrix_algorithm#Variants>`_
+
+        .. math::
+            R
+            =
+            \begin{pmatrix}
+                4 & 1 & 0 & 0 & 1 \\
+                1 & 4 & 1 & 0 & 0 \\
+                0 & 1 & 4 & 1 & 0 \\
+                0 & 0 & 1 & 4 & 1 \\
+                1 & 0 & 0 & 1 & 4
+            \end{pmatrix}
+            &=
+            \begin{pmatrix}
+                3 & 1 & 0 & 0 & 0 \\
+                1 & 4 & 1 & 0 & 0 \\
+                0 & 1 & 4 & 1 & 0 \\
+                0 & 0 & 1 & 4 & 1 \\
+                0 & 0 & 0 & 1 & 3
+            \end{pmatrix}
+            +
+            \begin{pmatrix}
+                1 & 0 & 0 & 0 & 1 \\
+                0 & 0 & 0 & 0 & 0 \\
+                0 & 0 & 0 & 0 & 0 \\
+                0 & 0 & 0 & 0 & 0 \\
+                1 & 0 & 0 & 0 & 1
+            \end{pmatrix}
+            \\
+            &
+            \\
+            &=
+            \begin{pmatrix}
+                3 & 1 & 0 & 0 & 0 \\
+                1 & 4 & 1 & 0 & 0 \\
+                0 & 1 & 4 & 1 & 0 \\
+                0 & 0 & 1 & 4 & 1 \\
+                0 & 0 & 0 & 1 & 3
+            \end{pmatrix}
+            +
+            \begin{pmatrix}
+                1 \\
+                0 \\
+                0 \\
+                0 \\
+                1
+            \end{pmatrix}
+            \begin{pmatrix}
+                1 & 0 & 0 & 0 & 1
+            \end{pmatrix}
+            \\
+            &
+            \\
+            &=
+            T + uv^t
+
+        We decompose :math:`R = T + uv^t`, where :math:`T` is a tridiagonal matrix, and
```

Comment:
Personally, although I've seen `T` mostly used for the transpose, I've also seen `t` quite a lot. But the main reason I chose `t`, is because I already used `T` for the `T`ridiagonal matrix.

Reply:
At first glance, I thought it was a hermitian transpose, that's all :)
It's fairly obvious that it isn't actually the case, though.